src/main/kotlin/com/openlattice/hazelcast/processors/RemoveDataExpirationPolicyProcessor.kt

```diff
@@ -0,0 +1,31 @@
+package com.openlattice.hazelcast.processors
+
+import com.kryptnostic.rhizome.hazelcast.processors.AbstractRhizomeEntryProcessor
+import com.openlattice.edm.EntitySet
+import java.util.UUID
+
+class RemoveDataExpirationPolicyProcessor() :
+        AbstractRhizomeEntryProcessor<UUID, EntitySet, EntitySet>() {
+
+    companion object {
+        private const val serialVersionUID = -6602384557982347L //TODO ????
+    }
+
+    override fun process(entry: MutableMap.MutableEntry<UUID, EntitySet>?): EntitySet {
```

Comment:
Update signature to:
`override fun process(entry: MutableMap.MutableEntry<UUID, EntitySet?>): EntitySet {`
specifically: 
`Entry<UUID, EntitySet?>` specifically